doc/library-contributor.rst

```diff
@@ -30,14 +30,19 @@ The ``neptune-release`` tool automatically collates release notes when making a
 release. To make this possible, contributors must follow the following
 guidelines.
 
-Major and Minor Releases
-........................
-
 When making a commit to a Neptune-versioned library, API-affecting changes
```

Comment:
Suggest:
```
The ``neptune-release`` tool automatically collates release notes when making a release. To make this possible, contributors must write release notes in a specific format (described below), and place them in the ``relnotes`` folder. When the corresponding branch is released, the files in that folder will be collated to form the notes for the release.

The following procedure should be followed:
... (as before)

Patch Releases
..............
For patch releases, a list of all fixed issues will additionally be appended to the release notes. The title of the bug fix pull request is used for this, so it is important to choose a clear and descriptive name for PRs containing bug fixes.
```

Reply:
When you write "as before" do you mean as before my PR or as IN my PR?

Reply:
In you PR. (Sorry, I was forgetting that you'd changed one line in that section.)